server/http.go

```diff
@@ -409,36 +437,66 @@ func (p *Plugin) handleStartMeeting(w http.ResponseWriter, r *http.Request) {
 		p.postAuthenticationMessage(req.ChannelID, userID, authErr.Message)
 		return
 	}
-
-	client, _, err := p.getActiveClient(user)
-	if err != nil {
-		p.API.LogWarn("Error getting the client", "err", err)
-		return
+	// topic
```

Comment:
why such vague comments either write complete sentences or don't write them at all
```suggestion

	// topic
```